slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -65,8 +65,9 @@ ::grpc::Status SLAMServiceImpl::GetMap(ServerContext *context,
         // paint map accordingly with or without the marker
         // included. Ticket: https://viam.atlassian.net/browse/DATA-657
         std::string jpeg_img = "";
+        bool marker_flag = request->include_robot_marker();
```

Comment:
[q] Will we have to implement adding the marker to the pointcloud as well? Did we do that for orbslam? If so, should we create a ticket for this?

Reply:
[opt] `pose_marker_flag` instead of `marker_flag`

Reply:
we did not include the marker for the pointcloud, in our spec we only wrote it for the jpeg endpoint